DuckDuckGo/AppDelegate+Waitlists.swift

```diff
@@ -51,12 +52,17 @@ extension AppDelegate {
 
         VPNWaitlist.shared.fetchInviteCodeIfAvailable { [weak self] error in
             guard error == nil else {
-#if !DEBUG
```

Comment:
Removed this since there's no reason this code shouldn't run in debug code any more. It was better this way early on, but now we can enable this.